api/dc.js

```diff
@@ -51,18 +85,32 @@ const dcPostBase = {
  */
 const dcGetByCharacter = {
   method: ['GET'],
-  path: '/dc/{character}',
+  path: '/dc/{characters}',
   options: {
     handler: (req, handlr) => {
-      const options = Object.assign({}, req.query, req.payload, req.params, { universe: 'dc' });
-      const config = api.handleConfig(options);
-      return api.handleRequest(config, handlr);
+      const options = Object.assign({}, req.query, req.payload, req.params, { universe });
+      let response;
+      if ('help' in options) {
```

Comment:
this is super nice. love being able to get help this way

Reply:
Thanks man. It's very Pythonic actually.